### ISSUE TITLE
Add prefix for CYB tokens

### ISSUE THREAD

**Issue by asadovka** [cybercongress/cyb-archeology#165]
As there are lots of CYB tokens (10^18) and it is not an easy task to distinguish all the digits we need a prefix for them.

Like in physical quantities:

CYB
KCYB
MCYB
GCYB
TCYB
PCYB

also we need to round such values (leave 2 digits for integer part and 2 for decimals):

##.## PREFIX_CYB

**Comment by mastercyb** [comment]
Please remove 2 decimals. Only whole numbers. Never use decimals for CYB

1 CYB is 1 CYB
1 KCYB is 1 000 CYB
1 MCYB is 1 000 000 CYB

So if you want to show 1.9 KCYB lets show 1900 CYB instead

It sucks to write, read and pronounce numbers with decimals. Also, decimals do not exist in our case

**Comment by hleb-albau** [comment]
10 937 863 831 CYB ~ 10.938 GCYB
10 937 863 831 CYB ~ 10 938  MCYB  
As for me, the first variant is more natural(as in every chain). In regular spech we alwaus say **"aroud 11 Gigacyb"**, rather than **"11k Megacyb"**.
It can be display option either.

**Comment by mastercyb** [comment]
I am going to create usability guidelines.  Please, do not use decimals

@hleb-albau You just confirmed my thesis that it is hard to parse, because you need to make calculations before pronouncing. 

My idea is exactly what you say. Instead of decimals we can round, etc...
